Formula/tarantool@1.10.rb

```diff
@@ -0,0 +1,60 @@
+class TarantoolAT110 < Formula
+  desc "In-memory database and Lua application server (1.10.4 LTS)"
+  homepage "https://tarantool.org/"
+  url "https://download.tarantool.org/tarantool/1.10/src/tarantool-1.10.4.1.tar.gz"
```

Comment:
That makes this a copy of the other formula right?

Reply:
Deleted.